partner_vat_unique/models/res_partner.py

```diff
@@ -24,5 +24,6 @@ def _check_vat_unique(self):
                 continue
             if record.same_vat_partner_id:
                 raise ValidationError(
-                    _("The VAT %s already exists in another partner.") % record.vat
+                    self.env._("The VAT %s already exists in another partner.")
```

Comment:
@lef-adhoc Please use translation function arguments instead.

`self.env._("The VAT %(vat)s already exists in another partner.", vat=record.vat)`

Reply:
Done!